### ISSUE TITLE
Error importing Mercator projection from CF

### ISSUE THREAD

**Issue by pgamez** [pyproj4/pyproj#461]
Hello,

I've found an issue running the method `CRS.from_cf()` on a mercator projection.
The method maps the keyword `standard_parallel` to `lat_1` instead of `lat_ts`. This code reproduces the issue and how to obtain the expected output:

```python
#!/usr/bin/env python3

import pyproj as pj

# Example data, according to:
#   http://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/cf-conventions.html#_mercator

kw = {
    'grid_mapping_name': 'mercator',
    'longitude_of_projection_origin': 10,
    'standard_parallel': 21.354,
    'false_easting': 0,
    'false_northing': 0,
}

crs = pj.CRS.from_cf(kw)
print(crs)

print('to proj4:')
print('\t', crs.to_proj4())

print('Fixed:')
srs = crs.srs.replace('+lat_1=', '+lat_ts=')
crs = crs.from_proj4(srs)
print('\t', crs.to_proj4())
```
This is the output:

```
+proj=merc +lat_1=21.354 +lon_0=10 +type=crs
to proj4:
	 +proj=merc +lon_0=10 +k=1 +x_0=0 +y_0=0 +datum=WGS84 +units=m +no_defs +type=crs
Fixed:
	 +proj=merc +lat_ts=21.354 +lon_0=10 +x_0=0 +y_0=0 +datum=WGS84 +units=m +no_defs +type=crs
```
As you can see, the proj4 string `+proj=merc +lon_0=10 +k=1 ...` does not match with the CF dict (note the value `k=1`). In the other hand, the last string `+proj=merc +lat_ts=21.354 +lon_0=10 ...` is correct.

I was using:
- Python v3.6.6
- pyproj v2.4.0

**Comment by pgamez** [comment]
BTW, there is no reference to the parameter `lat_1` at [proj4 docs](https://proj.org/operations/projections/merc.html), only `lat_ts`

**Comment by snowman2** [comment]
Thanks for the report and the research! It made for a quick fix in #463.
